src/gcs/gcsDrive.ts

```diff
@@ -208,25 +226,72 @@ export class GCSDrive implements Contents.IDrive {
     };
   }
 
-  async getDownloadUrl(localPath: string): Promise<string> {
-    throw 'Not Implemented';
+  async getDownloadUrl(
+    localPath: string,
+    options?: Contents.IFetchOptions
+  ): Promise<string> {
+    this.getFileContent(localPath, options);
+    return Promise.reject("File Download Successfully")
```

Comment:
Don't think we should reject the promise if we successfully downloaded.

Reply:
Done